File: app/app/Model/Chat/Response/ChatRoomSearchResponse.swift
struct ChatRoomSearchResponse: Codable, Identifiable {

    let chatRoomId: Int64
    let targetId: Int64
    let nickname: String
    let profileUrl: String?
    let lastMessage: String?
    let sortAt: String
    let unreadCount: Int
    let similarityScore: Double

    var id: Int64 { chatRoomId }
}

File: server/src/main/kotlin/com/pidulgi/server/chat/dto/response/ChatRoomSearchResponse.kt
package com.pidulgi.server.chat.dto.response

import java.time.LocalDateTime

data class ChatRoomSearchResponse(

    val chatRoomId: Long,
    val targetId: Long,
    val nickname: String,
    val profileUrl: String?,
    val lastMessage: String?,
    val sortAt: LocalDateTime,
    val unreadCount: Int,
    val similarityScore: Double,
)
